chatterbox/src/Main/MainArea/ChattingComponent/ChatArea/ChatArea.css

```diff
@@ -0,0 +1,78 @@
+.ChatArea{
+	width: 100%;
+	height: 100%;
+	background-color: white;
+	position: relative;
+}
+
+.ChatArea .writer-container{
+	width: 100%;
+	height: 230px;
+	position: absolute;
+	bottom: 0;
+}
+
+.ChatArea .writer-container .add-attachment{
```

Comment:
can you group these in a hierarchical structure?  they get a bit deep. not serious though, just nit-picking
```
.ChatArea {
     .writer-container {

     }
}
```

Reply:
Is that not scss syntax? does it work for normal css?